membership-common/conf/touchpoint.PROD.conf

```diff
@@ -66,6 +66,7 @@ touchpoint.backend.environments {
                     nationalDelivery="8a12999f8a268c57018a27ebddab1460"
                     digipack="2c92a0fb4edd70c8014edeaa4ddb21e7"
                     supporterPlus="8a12865b8219d9b4018221061563643f"
+                    tierThree="8a1295998ff2ec180190024b287b64c7"
```

Comment:
not for this PR but next time we're refactoring we can move these into string literals like we do in other codebases rather than trying to pick them out of the config file by key